modules/virt-configuring-vms-for-pci-passthrough.adoc

```diff
@@ -0,0 +1,7 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-configuring-pci-passthrough.adoc
+
+[id="virt-configuring-vms-for-pci-passthrough_{context}"]
+= Configuring virtual machines for PCI passthrough
+After the PCI devices have been added to the cluster, you can assign them to virtual machines. The PCI devices are now available as if they are physically connected to the virtual machines.
```

Comment:
Please delete this file now that it is not being used

Reply:
Adios to the file!